components/colorhandle/stories/template.js

```diff
@@ -0,0 +1,29 @@
+import { html } from 'lit-html';
+import { classMap } from 'lit-html/directives/class-map.js';
+
+import '../index.css';
+import '../skin.css';
+
+export const Template = ({
+  rootClass = "spectrum-ColorHandle",
+  customClasses = [],
+  isDisabled = false,
+  isFocused = false,
+  colorHandleStyle = "--spectrum-picked-color: rgba(255, 0, 0, 0.5); position: absolute; top: 50%; left: 50%;"
+  // ...globals
+}) => {
+  return html`
+  <div class=${classMap({
+    [rootClass]: true,
+    'is-disabled': isDisabled,
+    'is-focused': isFocused,
+    ...customClasses.reduce((a, c) => ({ ...a, [c]: true }), {}),
+  })}
+  ?disabled=${isDisabled}
+  ?focused=${isFocused}
+  style=${colorHandleStyle}>
```

Comment:
```suggestion
  style=${ifDefined(colorHandleStyle)}>
```

^ I'd wrap this in an ifDefined (and import ifDefined from lit at the top) and then this is good to go!